tests/utils/test_cases.py

```diff
@@ -242,3 +242,10 @@ def test_case_8() -> Collection:
                 "data-files/catalogs/" "planet-example-v1.0.0-beta.2/collection.json"
             )
         )
+
+    @staticmethod
+    def test_case_9() -> Catalog:
+        """Example catalog exposed by GeoServer"""
+        return Catalog.from_file(
+            TestCases.get_path("data-files/catalogs/test-case-9/catalog.json")
+        )
```

Comment:
```suggestion
```

I think it might be best to just use an existing test case and add a new link within the test method (see suggestion above). I'm not 100% sure because these pre-date me working on the project, but I think the test cases are mostly for tests where we want to make sure something works on a few different representative examples (like [here](https://github.com/stac-utils/pystac/blob/2d175d11223738778b54dfd37451134b0c291d44/tests/test_catalog.py#L248)), so I don't think we want to add new cases just for a single test (thoughts @lossyrob?).